types/react-dates/index.d.ts

```diff
@@ -266,9 +266,34 @@ declare namespace ReactDates {
         // internationalization props
         monthFormat?: string,
         phrases?: {
-            focusStartDate: string,
-            clearDates: string,
-            keyboardNavigationInstructions: string,
+            calendarLabel?: string,
```

Comment:
It seems to be a wrong Shape based on SingleDatePickerPhrases,
while here should be DayPickerPhrases  https://github.com/airbnb/react-dates/blob/master/src/defaultPhrases.js#L138,
can you please change?

Reply:
@Artur-A you are right. I have made the change.